lesson_07/homework_07.py

```diff
@@ -0,0 +1,175 @@
+# task 1
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+def multiplication_table(number):
+    # Initialize the appropriate variable
+    multiplier = 1
+
+    # Complete the while loop condition.
+    while multiplier <= 5:
+        result = number * multiplier
+        # десь тут помила, а може не одна
+        if  result > 25:
+            # Enter the action to take if the result is greater than 25
+            pass
+        print(str(number) + "x" + str(multiplier) + "=" + str(result))
+
+        # Increment the appropriate variable
+        multiplier += 1
+
+multiplication_table(3)
+# Should print:
+# 3x1=3
+# 3x2=6
+# 3x3=9
+# 3x4=12
+# 3x5=15
+
+
+# task 2
+"""  Написати функцію, яка обчислює суму двох чисел.
+"""
+def sum_two_numbers(a, b):
+    return a + b
+
+# Приклад використання:
+result = sum_two_numbers(3, 5)
+print(result)
+
+# task 3
+"""  Написати функцію, яка розрахує середнє арифметичне списку чисел.
+"""
+def average(numbers):
+    if len(numbers) == 0:
+        return 0  # Повертаємо 0, якщо список порожній
+    return sum(numbers) / len(numbers)
+numbers = [1, 2, 3, 4, 5]
+result = average(numbers)
+print("Середнє арифметичне:", result)
+
+# task 4
+"""  Написати функцію, яка приймає рядок та повертає його у зворотному порядку.
+"""
+def reverse_string(s):
+    return s[::-1]
+input_string = "Привіт Світ"
+reversed_string = reverse_string(input_string)
+print("Зворотний рядок:", reversed_string)
+
+# task 5
+"""  Написати функцію, яка приймає список слів та повертає найдовше слово у списку.
+"""
+def longest_word(words):
+    if not words:
```

Comment:
You solution is ok. But we have max function that can be customizable and so on.